.kitchen.docker.yml

```diff
@@ -18,7 +18,8 @@ platforms:
   driver:
     image: centos:7
     platform: rhel
-    run_command: /usr/lib/systemd/systemd
+    privileged: true
+    run_command: /sbin/init
```

Comment:
`/sbin/init` is still systemd, it's a symlink to the systemd binary.  i figured using the symlink's one less bit of variance between the platforms, and we don't have to worry about `/lib` vs `/usr/lib` differences between usr-merged and non-usr-merged platforms. it's also more in line with the system default (i.e. EL 7 hosts don't set init= kernel cmdline, so kernel's booting via the [default /sbin/init execve](https://github.com/torvalds/linux/blob/c8cdd2be213f0f5372287a764225665f1ac64e56/init/main.c#L966)). i don't know for sure if that matters, but i like sticking close to system defaults unless there's a need to deviate, so figured this would be preferable.

the privileged bit is a systemd thing too... much to my chagrine, systemd doesn't run correctly in docker without escalated privileges.